src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/ShellRenderer.cs

```diff
@@ -57,6 +58,9 @@ void InitializeComponent()
             {
                 _mainLayout = new Box(XForms.NativeParent);
                 _mainLayout.SetLayoutCallback(OnLayout);
+
+                InitializeNavigationDrawer();
```

Comment:
if no flyout items, we do not create navigation drawer

Reply:
Fixed.